File: perl/8_reverse_words.pl
#!/usr/bin/env perl

use strict;
use warnings;

open my $fh, '<', $ARGV[0];

my @lines = <$fh>;
for my $line ( @lines ) {
	chomp $line;
	print join " ", ( reverse( split( / /, $line ) ), "\n" );
}

close $fh;

File: python/8_reverse_words.py
#!/usr/bin/env python

import sys

with open(sys.argv[1], 'r') as fh:
    for line in fh.readlines():
        line = line.strip()
        if line == '':
            continue

        words = line.split(' ')
        words.reverse()
        print ' '.join(words)

